cloud_functions/token_refresh/test_main.py

```
@@ -0,0 +1,128 @@
+@pytest.fixture
+def mock_strava_response():
+    """Fixture for Strava token refresh response."""
+    return {
+        "access_token": "new_access_token",
+        "refresh_token": "new_refresh_token",
+        "expires_at": 1234567890,
+    }
+
+
+@pytest.fixture
+def mock_strava_secret():
```

Ruff (0.8.2)

Missing return type annotation for public function `mock_strava_secret`

(ANN201)